srcpkgs/pam_zfscrypt/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'pam_zfscrypt'
+pkgname=pam_zfscrypt
+version=0.2.0
+revision=1
+wrksrc="zfscrypt-${version}"
+build_style=gnu-makefile
+make_use_env=yes
+makedepends="pam-devel zfs-devel"
+depends="zfs pam"
```

Comment:
`zfs` dependency is pulled in via xbps-src, please remove this.

Reply:
Removed!